app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
```

Comment:
Consider adding a docstring at the beginning of the function to describe its purpose, parameters, and any exceptions it might raise. This will improve code readability and maintainability.